styles/components/navigation.css

```diff
@@ -0,0 +1,24 @@
+.navigation-bar {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.navigation-bar .nav-logo {
+  font-size: 1.3rem;
+  font-weight: 600;
+}
+
+.navigation-bar .gap {
```

Comment:
suggestion - you can use space between, or margin left auto, or space evenly instead of creating a gap class